pages/[date]/[room]/talks/[id].js

```diff
@@ -17,26 +22,50 @@ function TalkDetails() {
     day,
     date: talkDate,
     time,
-  } = data.find((talk) => talk.id === id);
+    isBookmarked,
+  } = talks.find((talk) => talk.id === id);
```

Comment:
```suggestion
 const {
    title,
    authors,
    abstract,
    session,
    day,
    date: talkDate,
    time,
    id: talkId,
    isBookmarked,
  } = talks.find((talk) => talk.id === id);
```

Reply:
This was the fundamental change, thanks!